lstchain/mc/sensitivity.py

```diff
@@ -74,6 +72,8 @@ def process_mc(simtelfile, dl2_file):
 
     gammaness = events.gammaness
 
+    gevents = events[events.mc_type==0]
+
     if events.iloc[0].mc_type==0:
```

Comment:
If you apply filter, then you want to work on `gevents` no? 😄

Reply:
Ups!, this remained from my tests on filtering, shouldn't have been in the commit, just removed it :')

Reply:
However, the code is still checking the first event only and therefore assumes that all particles are from the same type.
If you want to force the fact that files are from a single particle type, maybe add an assert about it somewhere. I feel like this kind of implicit assumption is error-prone for the future.

Reply:
I just included a way of filtering events, adding the type of particle as a function argument, so we can be sure that the resulting parameters belong to the desired type.